docs/doxygen/overviews/eventhandling.h

```diff
@@ -349,6 +349,33 @@ MyFrame::MyFrame()
 }
 @endcode
 
+You can also bind functions that are members of any class provided that the
+class publicly derives from wxTrackable from somewhere down the line and that
```

Comment:
This part seems just wrong to me, you can bind methods of any class, full stop. Why does it have to derive from `wxTrackable`?